TESTING.md

```diff
@@ -13,19 +13,23 @@ This library provides tools to help write tests for code that uses the following
 
 You can test against a temporary local Datastore by following these steps:
 
-1. Start the local Datastore emulator before running your tests using `LocalDatastoreHelper`'s `start` method. This will create a temporary folder on your computer and bind a port for communication with the local datastore. There are two optional arguments for `start`: project ID and consistency. The consistency setting controls the fraction of Datastore writes that are immediately visible in global queries.
+1. Start the local Datastore emulator before running your tests using `LocalDatastoreHelper`'s `create` and `start` methods. This will create a temporary folder on your computer and bind a port for communication with the local Datastore. There are two optional arguments for `create`: project ID and consistency. The consistency setting controls the fraction of Datastore writes that are immediately visible in global queries.
   ```java
-  // Use defaults for project ID and consistency
-  LocalDatastoreHelper helper = LocalDatastoreHelper.start();
+  // Use a placeholder project ID and the default consistency setting of 0.9
+  LocalDatastoreHelper helper = LocalDatastoreHelper.create();
   // or explicitly provide them
-  helper = LocalDatastoreHelper.start("my-project-id", 0.6);
+  helper = LocalDatastoreHelper.create("my-project-id", 0.6);
+  
+  helper.start(); // Starts the local Datastore emulator in a separate process
   ```
 
-2. In your program, create and use a `Datastore` object with the options given by the `LocalDatastoreHelper` instance.  For example:
+2. Create and use a `Datastore` object with the options given by the `LocalDatastoreHelper` instance.  For example:
   ```java
   Datastore localDatastore = helper.options().service();
   ```
   
+  Note that you must call `start` before `options()`; otherwise, you will see an `IOException` regarding a refused connection.
```

Comment:
Should we intercept this situation and throw a more informative `IllegalStateException` with custom message: "Cannot get service options before starting the emulator"?